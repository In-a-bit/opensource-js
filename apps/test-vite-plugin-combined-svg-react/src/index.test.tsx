import React from "react";
import { assert, expect, test } from "vitest";
import {
  ReactComponent as Icon,
  EnumStruct as IconType,
} from "./assets?svg-list-to-switcher";
import renderer from "react-test-renderer";
// Edit an assertion and save to see HMR in action

test("draw correct component html", () => {
  const component = renderer.create(<Icon iconName="UKR" />);

  expect(component).toMatchSnapshot();
});
