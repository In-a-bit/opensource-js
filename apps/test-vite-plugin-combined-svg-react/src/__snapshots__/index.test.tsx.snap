// Vitest Snapshot v1

exports[`draw correct component html 1`] = `
<svg
  fill="none"
  height={512}
  style={
    {
      "height": "24px",
      "width": "24px",
    }
  }
  viewBox="0 0 512 512"
  width={512}
  xmlns="http://www.w3.org/2000/svg"
>
  <g
    clipPath="url(#clip0_189_8074)"
  >
    <path
      d="M256 512C397.385 512 512 397.385 512 256C512 114.615 397.385 0 256 0C114.615 0 0 114.615 0 256C0 397.385 114.615 512 256 512Z"
      fill="#FFDA44"
    />
    <path
      d="M0 256C0 114.616 114.616 0 256 0C397.384 0 512 114.616 512 256"
      fill="#338AF3"
    />
  </g>
  <defs>
    <clipPath
      id="clip0_189_8074"
    >
      <rect
        fill="white"
        height={512}
        width={512}
      />
    </clipPath>
  </defs>
</svg>
`;
